grr/server/grr_response_server/gui/api_plugins/metadata.py

```diff
@@ -268,9 +263,82 @@ def _CreateMessageSchema(
       }
     )
 
+  def _CreateMapFieldSchema(
+      self,
+      field_descriptor: FieldDescriptor,
+      visiting: Set[str],
+  ) -> None:
+    """Creates the OpenAPI schema of a `protobuf.map` field type.
+
+    This method should generally not be called directly, but rather through
+    `_CreateSchema` which takes care of error-verifications and caching. The
+    OpenAPI Specification allows only maps with strings as keys, so, as a
+    workaround, we state the actual key type in the `description` fields of the
+    **schemas of the properties/parameters** that use this type (in order to be
+    displayed by documentation generation tools, see `_GetDescribedSchema`).
+    A `description` field is also added by this method in the schema definition
+    which will be added to the `Components Object` of the root `OpenAPI Object`
+    for more clarity when reading the generated description of the components.
+
+    Args:
+      field_descriptor: The protobuf `FieldDescriptor` associated with a
+        protobuf field.
+      visiting: A set of type names that are in the process of having their
+        OpenAPI schemas constructed and have their associated `_Create*Schema`
+        call in the current call stack.
+
+    Returns:
+      Nothing, the schema is stored in `self.schema_objs`.
+    """
+    if self.schema_objs is None:  # Check required by mypy.
+      raise AssertionError(
+        "The container of OpenAPI type schemas is not initialized."
+      )
```

Comment:
I would make it shorter to fit it the 80-character limit, e.g.: _"OpenAPI type schemas not initialized"_. In general, I believe that error messages should be rather concise (as long as we don't make them less informative), because then it is easier to follow the stacktrace (which usually is also line-wrapped at some point).

Reply:
Thank you for the detailed explanation! I made the change to the shorter message.